test/GameAreaTest.cpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file   GameAreaTest.cpp
+ * @brief  GameAreaクラスのテスト
+ * @author mutotaka0426
+ */
+
+#include "GameArea.h"
+#include <gtest/gtest.h>
+#include <gtest/internal/gtest-port.h>
+
+using namespace std;
+
+namespace etrobocon2022_test {
+
+  TEST(GameAreaTest, runLeftCourse)
+  {
+    bool isLeftCourse = true;
+    bool isLeftEdge = isLeftCourse;
+    int targetBrightness = 45;
+
+    testing::internal::CaptureStdout();  // 標準出力キャプチャ開始
+    GameArea::runGameArea(isLeftCourse, isLeftEdge, targetBrightness);  // 実行
+    string output = testing::internal::GetCapturedStdout();             // キャプチャ終了
+
+    bool expected = true;
+    // find("str")はstrが見つからない場合string::nposを返す
+    bool actual = output.find("Warning") == string::npos && output.find("Error") == string::npos;
+
+    EXPECT_EQ(expected, actual);  // WarningやErrorが出ていないかテスト
```

Comment:
`EXPECT_TRUE`というものが使えるようです([参考資料](http://opencv.jp/googletestdocs/primer.html#primer-basic-assertions))。
```suggestion
    // find("str")はstrが見つからない場合string::nposを返す
    bool actual = output.find("Warning") == string::npos && output.find("Error") == string::npos;

    EXPECT_TRUE(actual);  // WarningやErrorが出ていないかテスト
```